.github/workflows/test-indexer-debian.yml

```diff
@@ -0,0 +1,42 @@
+name: Test Wazuh Indexer upon upgrade - Debian
+on:
+  pull_request:
+    paths:
+      - 'stack/indexer/deb/debian/postrm'
+      - 'stack/indexer/deb/debian/postinst'
+      - 'stack/indexer/deb/debian/preinst'
+  workflow_dispatch:
+
+jobs:
+  Test-debian-indexer-upgrade:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download Wazuh Indexer release package
+        run: |
+          curl -s https://packages.wazuh.com/key/GPG-KEY-WAZUH | gpg --no-default-keyring --keyring gnupg-ring:/usr/share/keyrings/wazuh.gpg --import && chmod 644 /usr/share/keyrings/wazuh.gpg
+          echo "deb [signed-by=/usr/share/keyrings/wazuh.gpg] https://packages.wazuh.com/4.x/apt/ stable main" | sudo tee -a /etc/apt/sources.list.d/wazuh.list
+          sudo apt-get update
+
+      - name: Get the release version of Wazuh (all components)
+        run: echo "RELEASE_VERSION=$(jq -r 'map(select(.prerelease == false and .draft == false)) | .[] | .tag_name' <<< $(curl --silent https://api.github.com/repos/wazuh/wazuh/releases) | sed "s|v||g" | sort -rV | head -n 1)" >> $GITHUB_ENV
+
+      - name: Get the major and minor of the release version
```

Comment:
This can not be performed in a unique task using variables. Notice that the `RELEASE_VERSION` environment variable is used in the `MAJOR_MINOR` environment variable, and as the documentation explains the environment variables can be used in the subsequent task, but not in the same task where the variable has been defined. (https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable):

> You can make an environment variable available to any subsequent steps in a workflow job

A possible solution is to replace the `RELEASE_VERSION` environment variable with the `jq` command, but I think the whole command would be excessively long and illegible. Please reconsider keeping these two tasks separated.

`RELEASE_VERSION` is changed by `LATEST_STABLE_VERSION` following: https://github.com/wazuh/wazuh-packages/pull/1998#discussion_r1065758582